time="2025-09-20T12:08:54+02:00" level=warning msg="/home/gabriele/thesis-project/Agents-for-Vulnerable-Dockers-and-related-Benchmarks/dockers/CVE-2020-11652/openai/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion"
#1 [internal] load local bake definitions
#1 reading from stdin 719B done
#1 DONE 0.0s

#2 [internal] load build definition from Dockerfile
#2 transferring dockerfile: 370B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/library/python:3.8-slim
#3 DONE 1.1s

#4 [internal] load .dockerignore
#4 transferring context: 2B done
#4 DONE 0.0s

#5 [1/5] FROM docker.io/library/python:3.8-slim@sha256:1d52838af602b4b5a831beb13a0e4d073280665ea7be7f69ce2382f29c5a613f
#5 resolve docker.io/library/python:3.8-slim@sha256:1d52838af602b4b5a831beb13a0e4d073280665ea7be7f69ce2382f29c5a613f 0.0s done
#5 DONE 0.0s

#6 [internal] load build context
#6 transferring context: 186B done
#6 DONE 0.0s

#7 [2/5] RUN apt-get update &&     apt-get install -y gcc libzmq3-dev &&     pip install --no-cache-dir distro==1.5.0 &&     pip install --no-cache-dir salt==3000.2 pyzmq==19.0.0
#7 CACHED

#8 [3/5] COPY salt-master.conf /etc/salt/master
#8 CACHED

#9 [4/5] COPY entrypoint.sh /entrypoint.sh
#9 CACHED

#10 [5/5] RUN chmod +x /entrypoint.sh
#10 CACHED

#11 exporting to image
#11 exporting layers done
#11 exporting manifest sha256:a5a689493625880dda2c21c6bd5ff9c4b5b5bd1870c61e6fd6955c65e5c46de7 done
#11 exporting config sha256:efbd13882a274c315d04642a4b3d519b98bb135e0f067d4f6d8328fe8d418ba3 done
#11 exporting attestation manifest sha256:ee0dd02932f37b0cb0b54d5773ef45c076acdd50580f55af2272ccd5bf4fae4a 0.0s done
#11 exporting manifest list sha256:80c5d39e55c2be90d6b2c6e97565f5029d0b6f66b23844d3742d31528f136f72 done
#11 naming to docker.io/library/openai-salt-master:latest
#11 naming to docker.io/library/openai-salt-master:latest done
#11 unpacking to docker.io/library/openai-salt-master:latest
#11 unpacking to docker.io/library/openai-salt-master:latest 3.1s done
#11 DONE 3.2s

#12 resolving provenance for metadata file
#12 DONE 0.0s
 openai-salt-master  Built
 Network openai_default  Creating
 Network openai_default  Created
 Volume "openai_salt-cache"  Creating
 Volume "openai_salt-cache"  Created
 Volume "openai_salt-config"  Creating
 Volume "openai_salt-config"  Created
 Container openai-salt-master-1  Creating
 Container openai-salt-master-1  Created
 Container openai-salt-master-1  Starting
 Container openai-salt-master-1  Started


sudo docker logs 7822ae5cc8f0ef0b0c3fe69e880afaf8e161c4c180bb59d2d0967eee49bf08da --details
STDOUT: 
STDERR:  Traceback (most recent call last):
   File "/usr/local/bin/salt-master", line 8, in <module>
     sys.exit(salt_master())
   File "/usr/local/lib/python3.8/site-packages/salt/scripts.py", line 96, in salt_master
     import salt.cli.daemons
   File "/usr/local/lib/python3.8/site-packages/salt/cli/daemons.py", line 48, in <module>
     import salt.utils.parsers
   File "/usr/local/lib/python3.8/site-packages/salt/utils/parsers.py", line 28, in <module>
     import salt.config as config
   File "/usr/local/lib/python3.8/site-packages/salt/config/__init__.py", line 100, in <module>
     _DFLT_IPC_WBUFFER = _gather_buffer_space() * .5
   File "/usr/local/lib/python3.8/site-packages/salt/config/__init__.py", line 89, in _gather_buffer_space
     import salt.grains.core
   File "/usr/local/lib/python3.8/site-packages/salt/grains/core.py", line 40, in <module>
     from platform import _supported_dists
 ImportError: cannot import name '_supported_dists' from 'platform' (/usr/local/lib/python3.8/platform.py)


